src/patternfly/components/Page/page.scss

```diff
@@ -1,9 +1,14 @@
+// stylelint-disable
 $pf-c-page--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl", "2xl");
 $pf-c-page--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl", "2xl");
+// stylelint-enable
 
 // URL.com/guidelines#layout
 .pf-c-page {
   --pf-c-page--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+  --pf-c-page--inset: 0;
+  --pf-c-page--m-page-insets--inset: var(--pf-global--spacer--md);
+  --pf-c-page--m-page-insets--xl--inset: var(--pf-global--spacer--lg);
```

Comment:
I like that we have a higher order var for this, as it lets us use this other places - page sections, masthead, etc. But I don't know if this is entirely accurate. The page does have an inset without `.pf-m-page-insets`.

As you have it now, you'll need a class on a thing that doesn't use page insets to add the page inset. I think it might make more sense to be something like this - wdyt?

```
--pf-c-page--inset: var(--pf-global--spacer--md);
--pf-c-page--xl--inset: var(--pf-global--spacer--lg);

--pf-c-page__[thing]--m-page-insets--PaddingLeft: var(--pf-c-page--inset);
--pf-c-page__[thing]--m-page-insets--PaddingRight: var(--pf-c-page--inset);

@media (min-width: [xl breakpoint]) {
  --pf-c-page--inset: var(--pf-c-page--xl--inset);
}

.pf-c-page__[thing] {
  &.pf-m-page-insets {
    --pf-c-page__[thing]--PaddingLeft: var(--pf-c-page__[thing]--m-page-insets--PaddingLeft);
    --pf-c-page__[thing]--PaddingRight: var(--pf-c-page__[thing]--m-page-insets--PaddingRight);
  }
}
```